docker-compose.yaml

```diff
@@ -1,29 +1,27 @@
 version: '3.7'
 
 x-jellyfish-template: &jellyfish-template
-  image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-0.1.0-4a3173b}"
+  image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-0.1.0-5ef4452}"
   restart: unless-stopped
   environment: &jellyfish-environment
-    JF_INTEGRATED_TURN_IP: $EXTERNAL_IP
-    JF_INTEGRATED_TURN_LISTEN_IP: "0.0.0.0"
-    JF_SERVER_API_TOKEN: ${SERVER_API_TOKEN}
-    RELEASE_COOKIE: ${SERVER_API_TOKEN}
-    JF_NODES: "app@jellyfish1 app@jellyfish2"
-    JF_CHECK_ORIGIN: "${CHECK_ORIGIN:-false}"
-    BEAM_PORT: 9000
-    ELIXIR_ERL_OPTIONS: "-kernel inet_dist_listen_min $BEAM_PORT inet_dist_listen_max $BEAM_PORT"
+    JF_SERVER_API_TOKEN: ${JF_SERVER_API_TOKEN}
+    JF_CHECK_ORIGIN: "${JF_CHECK_ORIGIN:-false}"
+    JF_DIST_ENABLED: "${JF_DIST_ENABLED:-true}"
+    JF_DIST_NODES: "app@jellyfish1 app@jellyfish2"
+    JF_WEBRTC_TURN_IP: $EXTERNAL_IP
+    JF_WEBRTC_TURN_LISTEN_IP: "0.0.0.0"
 
 services:
   jellyfish1:
     <<: *jellyfish-template
     container_name: jellyfish1
     environment:
       <<: *jellyfish-environment
-      JF_INTEGRATED_TURN_PORT_RANGE: "50000-50500"
-      JF_INTEGRATED_TCP_TURN_PORT: "49999"
-      JF_HOST: "${JELLYFISH1_ADDRESS:-${JELLYFISH1_IP:-$EXTERNAL_IP}}"
-      RELEASE_NODE: app@jellyfish1
       JF_PORT: 5002
+      JF_HOST: "${JF1_HOST:-${JF1_IP:-$EXTERNAL_IP}:$JF_PORT}"
```

Comment:
I think this is a typo because we didn't add `JF_PORT` in `.env.example`, so a user won't have this env set and line before it is set inside the docker container, and by `$` sign, we refer to the environment variables on the host. I will fix it in #22.

Reply:
This commit fixes these problems: https://github.com/jellyfish-dev/jellyfish_videoroom/pull/22/commits/65bb843660c8ba948dda599bd2eade30bb72afee